scenarios/kubernetes_kops_aws.py

```diff
@@ -198,6 +198,9 @@ def main(args):
 
     cmd.append(kubekins(args.tag))
 
+    if args.kops_args:
```

Comment:
This is pretty silly to fail line.

Feel free to set L68 to `too-many-locals,too-many-branches`